pkg/vm/vm_test.go

```diff
@@ -2435,7 +2435,9 @@ func makeProgram(opcodes ...opcode.Opcode) []byte {
 
 func load(prog []byte) *VM {
 	vm := New()
-	vm.LoadScript(prog)
+	if len(prog) != 0 {
```

Comment:
What's the use case?

Reply:
https://github.com/neo-project/neo-vm/blob/master/tests/neo-vm.Tests/Tests/Others/OtherCases.json#L22

Reply:
Well, you'd better write it in commit message then. Looks weird anyway.